packages/db/postgres/functions.sql

```diff
@@ -19,6 +19,11 @@ CREATE OR REPLACE FUNCTION json_arr_to_json_element_array(_json json)
   RETURNS json[] LANGUAGE sql IMMUTABLE PARALLEL SAFE AS
 'SELECT ARRAY(SELECT * FROM json_array_elements(_json))';
 
+-- transform a JSON array property into an array of SQL jsonb elements
+CREATE OR REPLACE FUNCTION json_arr_to_jsonb_element_array(_json json)
```

Comment:
Isn't this a duplicate of [json_arr_to_json_element_array](https://github.com/web3-storage/web3.storage/pull/1305/files#diff-bf4100e6931ed75d8fbe33d506c141af632f120f4df66bfd0ff9c92d80da40c6R18)?